tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -714,28 +310,362 @@ public function doSomething($foo)
+                     * @Template(foobar=21)
+                     * @param Entity $foo
+                     */
+                    public function doSomething($foo)
+                    {
+                        $bar = $foo->toArray();
+                        /** @var ArrayInterface $bar */
+
+                        return function () use ($bar, $baz) {};
+                    }
+                }
+                EOF,
+        ];
+
+        yield 'with different cases' => [
+            <<<'EOF'
+                The normal
+                use of this fixer
+                should not change this sentence nor those statements below
+                use Zoo\Baz;
+                use abc\Bar;
+
+                <?php
+
+                use abc\Bar;
+                use Zoo\Baz;
+
+                class Test
+                {
+                }
+                EOF,
+            <<<'EOF'
+                The normal
+                use of this fixer
+                should not change this sentence nor those statements below
+                use Zoo\Baz;
+                use abc\Bar;
+
+                <?php
+
+                use Zoo\Baz;
+                use abc\Bar;
+
+                class Test
+                {
+                }
+                EOF,
+        ];
+
+        yield 'without uses' => [
+            <<<'EOF'
+                <?php
+
+                $c = 1;
+                EOF,
+        ];
+
+        yield 'with trailing digits' => [
+            <<<'EOF'
+                <?php
+
+                use abc\Bar;
+                use abc2\Bar2;
+                use xyz\abc\Bar6;
+                use xyz\abc2\Bar7;
+                use xyz\xyz\Bar4;
+                use xyz\xyz\Bar5;
+
+                class Test
+                {
+                }
+                EOF,
+            <<<'EOF'
+                <?php
+
+                use abc2\Bar2;
+                use abc\Bar;
+                use xyz\abc2\Bar7;
+                use xyz\abc\Bar6;
+                use xyz\xyz\Bar4;
+                use xyz\xyz\Bar5;
+
+                class Test
+                {
+                }
+                EOF,
+        ];
+
+        yield 'with imports only' => [
+            <<<'EOF'
+                <?php
+
+                use Aaa;
+                use Bbb;
+                EOF,
+            <<<'EOF'
+                <?php
+
+                use Bbb;
+                use Aaa;
+                EOF,
+        ];
+
+        yield 'with close tag' => [
+            '<?php
+                use A\C1;
+                use A\D?><?php use B\C2; use E\F ?>',
+            '<?php
+                use A\C1;
+                use B\C2?><?php use A\D; use E\F ?>',
+        ];
+
+        yield 'with comments' => [
+            '<?php
+                use A\C1 /* A */;
+                use /* B */ B\C2;',
+            '<?php
+                use /* B */ B\C2;
+                use A\C1 /* A */;',
+        ];
+
+        yield [
```

Comment:
Missing the case name, let's keep consistency and provide a description for all of them.